index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id= "navbar" style ="margin-top: 25px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in a separate CSS file.

Reply:
- [ ] [REQUIRED] Please, mention the reason for the requested change (e.g: Please do not use inline styles. Keep your style definitions in a separate CSS file `to follow best practices css coding and to have the ability to attach the css file to multiple html files at the same time and save time and resources`

Reply:
Dear @OmarRamoun, 

Thank you for your kind reply. My question is should I edit the comment that I previously added or add a new comment?

Thanks,